### ISSUE TITLE
calicoctl fails to run and just prints help text if ip6tables is missing

### ISSUE THREAD

**Issue by tomdee** [projectcalico/calico-docker#531]
```
./calicoctl node
calicoctl

Override the host:port of the ETCD server by setting the environment variable
ETCD_AUTHORITY [default: 127.0.0.1:4001]

Usage: calicoctl <command> [<args>...]

    status            Print current status information
    node              Configure the main calico/node container and establish Calico networking
    container         Configure containers and their addresses
    profile           Configure endpoint profiles
    endpoint          Configure the endpoints assigned to existing containers
    pool              Configure ip-pools
    bgp               Configure global bgp
    ipam              Configure IP address management
    checksystem       Check for incompatibilities on the host system
    diags             Save diagnostic information
    version           Display the version of calicoctl
    config            Configure low-level component configuration

See 'calicoctl <command> --help' to read about a specific subcommand.
```

By commenting out the exception handling code I found this
```
/code # ./calicoctl node
Traceback (most recent call last):
  File "<string>", line 104, in <module>
  File "/code/build/calicoctl/out00-PYZ.pyz/calico_ctl.node", line 194, in node
  File "/code/build/calicoctl/out00-PYZ.pyz/calico_ctl.node", line 217, in node_start
  File "/code/build/calicoctl/out00-PYZ.pyz/calico_ctl.checksystem", line 67, in check_system
  File "/code/build/calicoctl/out00-PYZ.pyz/calico_ctl.checksystem", line 104, in _check_kernel_modules
  File "/code/build/calicoctl/out00-PYZ.pyz/sh", line 776, in _create
sh.CommandNotFound: ip6tables
```

**Comment by alexhersh** [comment]
Moved the `ip6tables` command object creation into the exception handler. It should complain if it can't find ip6tables now.

**Comment by alexhersh** [comment]
Resolved in #544
